godot-ffi/src/godot_ffi.rs

```diff
@@ -94,6 +94,53 @@ pub unsafe trait GodotFfi {
     unsafe fn move_return_ptr(self, dst: sys::GDExtensionTypePtr, call_type: PtrcallType);
 }
 
+/// Marks a type as having a nullable counterpart in Godot.
+///
+/// This trait primarely exists to implement GodotFfi for `Option<Gd<T>>` which is not possible
+/// due to Rusts orphan rule. The rule also enforces better API design, though. `godot_ffi` should
+/// not concern it self with the details of how Godot types work and merly defines the FFI abstraction.
+/// By having a marker trait for nullable types we can provide a generic implementation for all
+/// compatible types without having to know which types they are.
```

Comment:
A few typos and missing commas:
```suggestion
/// This trait primarily exists to implement GodotFfi for `Option<Gd<T>>`, which is not possible
/// due to Rusts orphan rule. The rule also enforces better API design, though. `godot_ffi` should
/// not concern itself with the details of how Godot types work and merely defines the FFI abstraction.
/// By having a marker trait for nullable types, we can provide a generic implementation for all
/// compatible types, without knowing their definition.
```

Also, this sounds a bit like there are dozens of types that can be supported by the trait, while in reality it's just 1 for the moment.